src/engraving/dom/rest.cpp

```diff
@@ -274,10 +274,10 @@ EngravingItem* Rest::drop(EditData& data)
     }
     case ElementType::SYMBOL:
     case ElementType::IMAGE:
-        e->setParent(this);
+        e->setTrack(track());
+        e->setParent(segment());
         score()->undoAddElement(e);
         return e;
```

Comment:
It seems we can remove this lines, to let these types be handled by `ChordRest::drop`. Would mean less code duplication :)